scst/include/scst.h

```diff
@@ -1748,6 +1748,7 @@ struct scst_tgt {
 	char *tgt_comment;
 
 	uint16_t rel_tgt_id;
+	uint16_t tg_id; /* Only used during TYPE_DISK INQUIRY EVPD=0x83 */
```

Comment:
I think it's better to move `tg_id` to scst_cmd because it is calculated per cmd and not per scst_tgt device. For example, you could place this here:

```
/*
 * SCST command, analog of I_T_L_Q nexus or task
 */
struct scst_cmd {
        ...
        uint32_t tgt_sn; /* SN set by target driver (for TM purposes) */
+
+       uint16_t tg_id; /* Only used during TYPE_DISK INQUIRY EVPD=0x83 */
        ...
};
```